app/code/Magento/Tax/view/frontend/templates/order/tax.phtml

```diff
@@ -26,7 +26,7 @@
                 <td <?= /* @escapeNotVerified */ $block->getLabelProperties() ?>>
                     <?= $block->escapeHtml($title) ?>
                     <?php if (!is_null($percent)): ?>
-                        (<?= (float)$percent ?>%)
+                        (<?= /* @escapeNotVerified */ $_order->formatPrice((float)$percent) ?>%)
```

Comment:
Why not verified here?

Reply:
@orlangur  because it was reject in Travis CI process due to XSS vulnerability.

Reply:
@Shubham-Webkul there should be another annotation, like `noEscape`, to indicate escaping is not needed and logic was verified.